doc/dev/mgmt/tests.md

```diff
@@ -222,35 +170,13 @@ class ExampleResourceGroupTestCase(AzureMgmtTestCase):
 ```
 
-This simple test creates a resource group and checks that its name
-is assigned correctly.
+This simple test creates a resource group and checks that its name is assigned correctly.
 
 Notes:
-
-1.  This test inherits all necessary behavior for HTTP recording and playback
-    described previously in this document
-    from its `AzureMgmtTestCase` superclass.
-    You don't need to do anything special to implement it.
-2.  The `get_resource_name()` helper method of `AzureMgmtTestCase`
-    creates a pseudorandom name based on the parameter
-    and the names of the test file and method.
-    This ensures that the name generated is the same for each run
-    of the same test, thereby ensuring reproducability,
-    but prevents name collisions if the tests are run live
-    and the same parameter is used from several different tests.
-3.  The `create_mgmt_client()` helper method of `AzureMgmtTestCase`
-    creates a client object using the credentials
-    from `mgmt_settings_fake.py` or `mgmt_settings_real.py` as appropriate,
-    with some checks to make sure it's created successfully
-    and cause the unit test to fail if not.
-    You should use it for any clients you create.
-4.  Note that this test cleans up the resource group it creates!
-    If you create resources yourself as part of the test,
-    make sure to delete them afterwards.
-    But if you need something like a resource group
-    as a prerequisite for what you're actually trying to test,
-    you should use a "preparer" as demonstrated in the following two examples.
-    Preparers will create and clean up helper resources for you.
+1. This test inherits all necessary behavior for HTTP recording and playback described previously in this document from its `AzureMgmtTestCase` superclass. You don't need to do anything special to implement it.
+2. The `get_resource_name()` helper method of `AzureMgmtTestCase` creates a pseudorandom name based on the parameter and the names of the test file and method. This ensures that the name generated is the same for each run of the same test, ensuring reproducability and preventing name collisions if the tests are run live and the same parameter is used from several different tests.
+3. The `create_mgmt_client()` helper method of `AzureMgmtTestCase` creates a client object using the credentials from `mgmt_settings_fake.py` or `mgmt_settings_real.py` as appropriate, with some checks to make sure it's created successfully and cause the unit test to fail if not. You should use it for any clients you create.
+4. Note that this test cleans up the resource group it creates! If you create resources yourself as part of the test, make sure to delete them afterwards. But if you need something like a resource group as a prerequisite for what you're actually trying to test, you should use a "preparer" as demonstrated in the following two examples. Preparers will create and clean up helper resources for you.
```

Comment:
I understand what this sentence is trying to say:
>Note that this test cleans up the resource group it creates! If you create resources yourself as part of the test, make sure to delete them afterwards.

But on first read, it seems contradictory: i.e. the tests delete your resources. make sure you delete your resources. I think doing something like
>While the test cleans up the resource group it creates, you still need to manually delete any resources you've created independent of the test framework

would make it a little less confusing (at least for me).